l3kernel/l3fp.dtx

```diff
@@ -658,6 +658,137 @@
 %   the \meta{code} should make use of the \meta{tl~var}.
 % \end{function}
 %
+% \subsection{Symbolic expressions}
+%
+% Floating point expressions support variables: these are local-only constructs.
```

Comment:
Does that mean that `\fp_new_variable:n` will only initialise the variable on a local scope? If so, this breaks with `expl3`-conventions of other modules! If not, the wording should be adjusted.